charts/consul/test/terraform/gke/main.tf

```diff
@@ -29,7 +29,7 @@ resource "google_container_cluster" "cluster" {
     machine_type = "e2-standard-4"
   }
   pod_security_policy_config {
-    enabled = true
+    enabled = false
   }
```

Comment:
nit: we can just remove this config altogether since it's not enabled by default. Another thing here is that we can switch the google provider regular instead of beta.

Reply:
can do!